Userland/Demos/CatDog/SpeechBubble.cpp

```diff
@@ -11,11 +12,21 @@
 #include <LibGUI/Painter.h>
 #include <LibGfx/Palette.h>
 
-static Array<StringView, 3> messages = {
+static Array<StringView, 3> default_messages = {
     "It looks like you're trying to debug\na program. Would you like some help?"sv,
     "It looks like you're trying to shave\na yak. Would you like some help?"sv,
     "Well Hello Friend!"sv,
 };
+static Array<StringView, 3> pixel_paint_messages = {
+    "It looks like you're creating art.\nWould you like some help?"sv,
+    "It looks like you're making a meme\nfor Discord. :yakthonk:"sv,
```

Comment:
Could we use our custom Unicode Yak emojis here? (https://serenityos.net/~xexxa/10CD)

Reply:
I totally forgot, however maybe it's funnier this way. Let's have others chip in first.

Reply:
I think it might look better with our custom fonts